src/resources/UsageAnalytics/Read/DataHealth/DataHealthInterfaces.ts

```diff
@@ -310,3 +310,80 @@ export interface DataHealthGetTrackingIdsResponse {
      */
     values: DataHealthFacetValue[];
 }
+
+export interface DataHealthGetEventProblemsParams extends OrganizationParamParts, TimeRangeParamParts, Paginated {
+    /**
+     * A set of event types for which the events will be returned.
+     * If not specified, events are not filtered on event types.
+     */
+    eventType?: string[];
+    /**
+     * A set of search hubs for which the events will be returned.
+     * If not specified, events are not filtered on search hub.
+     */
+    searchHub?: string[];
+    /**
+     * The tracking id to filter by when getting groups listing.
+     * If not specified, events are not filtered on tracking id.
+     */
+    trackingId?: string[];
+}
+export interface EventProblemsResponse extends PaginatedResponse {
+    /**
+     * List of the event problems.
+     */
+    problems: EventProblemsResponseItem[];
+}
+
+export interface EventProblemsResponseItem {
+    /**
+     * A set of event types for which the events will be returned.
+     * If not specified, events are not filtered on event types.
+     */
+    eventType: string;
+    /**
+     * The tracking id to filter by when getting groups listing.
+     * If not specified, events are not filtered on tracking id.
+     */
+    trackingId: string;
+    /**
+     * A set of search hubs for which the events will be returned.
+     * If not specified, events are not filtered on search hub.
+     */
+    searchHub: string;
+    /**
+     * Number of failed events.
+     */
+    failedEvents: number;
+    /**
+     * Number of total events.
+     */
+    totalEvents: number;
+    /**
+     * List of the validation errors.
+     */
+    validationErrors: ValidationErrors[];
+    /**
+     * Events sampled from the problematic events.
```

Comment:
I don't understand what this means 😅

Can you explain, and then I can suggest a modification to the docs?

Reply:
So the issue is that the call is called "getEventProblems", but the "problems" are validation errors. The call might as well be called "getValidationError" :sweat_smile:

Reply:
OK, but what are the "sample events" ? Is it an incomplete list of event IDs that encountered the validation errors?

Reply:
Yea it's a list of exemple event that have the validation problem.